src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -273,6 +284,34 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
 
             return searchOptions;
 
+            IEnumerable<IncludeExpression> ParseIncludeIterateExpressions(SearchParams searchParams, bool reversed)
+            {
+                var iterateModifiers = reversed ? RevIncludeIterateModifiers : IncludeIterateModifiers;
+
+                return searchParams.Parameters
+                .Where(p => p != null && iterateModifiers.Where(m => string.Equals(p.Item1, m, StringComparison.OrdinalIgnoreCase)).Any())
+                .Select(p =>
+                {
+                    ResourceType parsedIncludeResourceType = ResourceType.DomainResource;
```

Comment:
No, it will not compile without it because of this if statement:

`if (!string.IsNullOrWhiteSpace(includeResourceType) &&
     !Enum.TryParse(includeResourceType, out parsedIncludeResourceType))
     {
            throw new ResourceNotSupportedException(includeResourceType);
      }